ch_pipeline/core/telescope.py

```diff
@@ -194,14 +200,28 @@ def _finalise_config(self):
 
     # Tweak the following two properties to change the beam width
     @property
-    def fwhm_e(self):
+    def fwhm_ex(self):
         """Full width half max of the E-plane antenna beam."""
```

Comment:
```suggestion
        """Full width half max of the E-plane antenna beam for the X polarization."""
```
and likewise for the other 3 properties.